source/end-user-guide/preferences/connect-multiple-workspaces.rst

```diff
@@ -17,39 +17,58 @@ Add a server
 
 .. tab:: Web/Desktop
 
-  The **Server** list is located in the top left corner of the window and displays all servers available. Drag to reorder the servers in the list. You can also navigate the server options using :doc:`keyboard shortcuts </end-user-guide/collaborate/keyboard-shortcuts>`. 
+  The **Server** list is located in the top left corner of the window and displays all servers available. Drag to reorder the servers in the list. You can also navigate the server options using :doc:`keyboard shortcuts </end-user-guide/collaborate/keyboard-shortcuts>`.
 
   .. note::
 
     If you're using the desktop app prior to release v5.0, individual servers display as separate tabs at the top of the window instead of the top left corner of the window as a list, and servers are managed by going to **… > File > Settings** on Windows and **Mattermost > Preferences** on Mac.
-  
+
   1. Select **Add a server**.
 
     .. image:: ../../images/desktop-server-add.png
       :alt: Connect the desktop app to a Mattermost Server using options located in the top left corner of the Mattermost screen.
 
   2. Enter the server URL. Server URLs must begin with either ``http://`` or ``https://``.
-  3. Enter the server's Display Name.
 
     .. tip::
 
       Can't find your Mattermost server URL? Ask your company’s IT department or your Mattermost system admin for your organization’s **Mattermost Site URL**. It’ll look something like ``https://example.com/company/mattermost``, ``mattermost.yourcompanydomain.com``, or ``chat.yourcompanydomain.com``. These URLs could also end in ``.net``.
 
-  4. Select **Add**.
+  3. Enter the server's Display Name.
+  4. Optionally, enter a **Pre-authentication Secret** if required by your server configuration.
```

Comment:
```suggestion
  4. If prompted, enter a **Pre-authentication Secret**. This is an additional security measure that some organizations use to control access to the Mattermost server. Your system admin can provide you with the **Pre-authentication Secret** value if required.
```

Reply:
This is not something the server is going to preemptively ask for. Users can try and connect to a server protected with a pre-authentication secret. Users must manually toggle the advanced settings section and enter the secret. The "If prompted" here seems confusing to me, but 0/5

Reply:
Thank you! I've updated my inline feedback. If you agree that it's accurate and complete, we can use the same language for line 63.